### ISSUE TITLE
instances from geometry nodes are not exported

### ISSUE THREAD

**Issue by xyning** [KhronosGroup/glTF-Blender-IO#1537]
**Describe the bug**
* as title

**To Reproduce**
1. create an empty scene and make some instances with gn
2. export the scene to gltf with `Apply Modifiers`
3. got an empty gltf file

**Expected behavior**
* all instances are exported

**Screenshots**
![スクリーンショット 2022-01-16 205019](https://user-images.githubusercontent.com/13201009/149660901-31fbcb38-a8ef-4b58-9244-6232e7e4d925.jpg)

**.blend file/ .gltf**
If applicable, a zipped folder containing a .blend file for exporter issues and a .gltf file for importer issues. Please take care to include all resources, such as textures in the .zip file.

**Version**
 - OS: windows
 - Blender Version: 3.0.0 

**Additional context**

**Comment by julienduroure** [comment]
I confirm that current API can't evaluate depsgraph for GN.
Settings "Blender label", until this is solved in Blender

**Comment by julienduroure** [comment]
I created this task on blender tracker:
https://developer.blender.org/T94953

**Comment by scurest** [comment]
You probably know this already, but I'll point out that a Realize Instances node will turn instances into regular geometry that gets exported correctly (though inefficiently) in any exporter.

**Comment by julienduroure** [comment]
@scurest
Yes, sure. But I hope that current work on geometry node (core blender), made it possible without this node.
But thanks, this is always great to say it again. Maybe this is something @xyning didn't know

**Comment by xyning** [comment]
yes that *should* work.. but actually not. [another blender bug](https://developer.blender.org/T85962) makes the `Realize Instances` node removing all UVs, which is unacceptable for exporting
currently the only(?) way to export them is to manually breakdown instances into thousands of objects by the `Make Instances Real` operator, then join them.. weird and laggy workflow
this is not really related to the exporter tho, thanks anyway

**Comment by Enlumis** [comment]
@xyning Been really searching on that issue for bit now..
i am trying to export my geo node and keep instance UVs, i realized that "realize Instances" node is removing UVs and that is real sad lol because that would have been the perfect solution..
i am also trying the "Make Instances Real" method but when blender breaks up the instances to a single objects it seems to also bring over all the other instances within the created instance, so i end up with thousands of instances in my supposedly single instance and it also recreate the geo modifier for each instances/single object.

So basically none of the solution works, really bangging my head against the wall for this problem..

**Comment by takahirox** [comment]
FYI: I'm writing an addon that allows to export instanced geometries created with geometry nodes as glTF + EXT_mesh_gpu_instancing extension https://github.com/takahirox/glTF-Blender-IO-EXT-mesh-gpu-instancing It started to work.

**Comment by feuerste** [comment]
Looking forward to having #1786 landed.
